src/compiler.ts

```diff
@@ -7061,13 +7075,13 @@ export class Compiler extends DiagnosticEmitter {
       calcValue, // also tees getValue to tempLocal
       false
     );
+
     this.currentType = tempLocal.type;
     currentFunction.freeTempLocal(tempLocal);
     var nativeType = tempLocal.type.toNativeType();
-    return module.createBlock(null, [
-      setValue,
-      module.createGetLocal(tempLocal.index, nativeType)
-    ], nativeType); // result of 'x++' / 'x--' might overflow
+    var getLocal   = module.createGetLocal(tempLocal.index, nativeType);
```

Comment:
This var seems unnecessary

Reply:
Removed this. And revert back to original